orca/split.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+
+def split_url(url, size):
+    start_end_format = re.compile(r"(([a-z]+)\[(\d*)(:\d*){0,1}:(\d*)\])")
+    var_time, var, start, stride, end = start_end_format.findall(url)[0]
+    start = int(start)
+    end = int(end)
+
+    # Server error reports dataset size = dataset.nbytes/2
+    b = size / 2
+    # Request limit is 5e+8 bytes
+    chunks = -(-b // 5e8)  # take ceiling
```

Comment:
Why are we throwing negatives in here?

Reply:
Ah ok:
```
>>> v = b // 500
>>> v
100
>>> v = -(-b // 500)
>>> v
101
```
I should have read the comment more closely. I just assumed `//` was a ceiling operator but it is for floor. While what you have is fine we could easily use `math.ceil` to make this a bit more readable (you also wouldn't need the comment anymore).